src/viam/examples/modules/complex/client.cpp

```diff
@@ -38,14 +37,19 @@ int main() {
     std::string address(uri);
     Options options(1, opts);
 
+    // Register custom gizmo and summation API so robot client can access resources
+    // of that type from the server.
+    Registry::register_resource(Gizmo::static_api(), Gizmo::resource_registration());
+    Registry::register_resource(Summation::static_api(), Summation::resource_registration());
```

Comment:
Users have to manually register custom APIs in their client _as well_ as their servers (not registering here means that the client resource manager cannot emplace a gizmo or summation client). This is normally not necessary as resource registration for regular APIs happens at runtime. I've written a note in https://viam.atlassian.net/browse/RSDK-5206 to investigate alternatives to this strategy.